app-modules/portal/database/migrations/2024_09_11_120345_data_migrate_gdpr_banner_text.php

```diff
@@ -0,0 +1,108 @@
+      service, where the service provides users with access to any substantial set of
+      the features or functionality of the software.
+    - You may not move, change, disable, or circumvent the license key functionality
+      in the software, and you may not remove or obscure any functionality in the
+      software that is protected by the license key.
+    - You may not alter, remove, or obscure any licensing, copyright, or other notices
+      of the licensor in the software. Any use of the licensor’s trademarks is subject
+      to applicable law.
+    - Canyon GBS LLC respects the intellectual property rights of others and expects the
+      same in return. Canyon GBS™ and Aiding App™ are registered trademarks of
+      Canyon GBS LLC, and we are committed to enforcing and protecting our trademarks
+      vigorously.
+    - The software solution, including services, infrastructure, and code, is offered as a
+      Software as a Service (SaaS) by Canyon GBS LLC.
+    - Use of this software implies agreement to the license terms and conditions as stated
+      in the Elastic License 2.0.
+
+    For more information or inquiries please visit our website at
+    <https://www.canyongbs.com> or contact us via email at legal@canyongbs.com.
+
+</COPYRIGHT>
+*/
+
+use Illuminate\Support\Facades\DB;
+use Illuminate\Database\Migrations\Migration;
+
+return new class () extends Migration {
+    public function up(): void
+    {
+        try {
+            DB::beginTransaction();
+
+            $oldGDPRBannerText = DB::table('settings')
+                ->where('group', 'portal')
+                ->where('name', 'gdpr_banner_text')
+                ->first();
+
+            if ($oldGDPRBannerText) {
+                DB::table('settings')->where('group', 'portal')
+                    ->where('name', 'cookie_gdpr_banner_text')
+                    ->update([
+                        'payload' => [
+                            'type' => 'doc',
+                            'content' => [
+                                [
+                                    'type' => 'paragraph',
+                                    'attrs' => [
+                                        'textAlign' => 'start',
+                                    ],
+                                    'content' => [
+                                        [
+                                            'type' => 'text',
+                                            'text' => json_decode($oldGDPRBannerText->payload),
+                                        ],
+                                    ],
+                                ],
+                            ],
+                        ],
+                    ]);
+            }
+
+            DB::commit();
+        } catch (Exception $error) {
+            DB::rollBack();
+
+            throw $error;
+        }
+    }
+
+    public function down(): void
```

Comment:
This wouldn't actually work in Production as it wouldn't set the value back to what it was if it had been modified from the default. But because we do not use the `downs` in Production and I want to get rid of them soon this should be fine for now